travis-setup.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -euo pipefail
+
+# Sets up travis-ci environment for testing bioconda-utils.
+
+if [[ $TRAVIS_OS_NAME = "linux" ]]
+then
+    tag=Linux
+else
+    tag=MacOSX
+fi
+curl -O https://repo.continuum.io/miniconda/Miniconda3-latest-${tag}-x86_64.sh
+sudo bash Miniconda3-latest-${tag}-x86_64.sh -b -p /anaconda
+sudo chown -R $USER /anaconda
+export PATH=/anaconda/bin:$PATH
+
+# TODO: add remaining pip reqs to conda-forge
+conda install -y --file conda-requirements.txt --channel bioconda --channel conda-forge --channel anaconda
```

Comment:
I don't think so. But you can install conda-build into the root env, and then instantiate environment.yml. However, this would also mean having two things. Let's keep it like this and try to add the pip packages to conda-forge at some point.